utils/ANLEra_help.cfg

```diff
@@ -43,6 +43,16 @@
     [/message]
 #enddef
 
+#define ANL_HELP_FORESTS
+    [message]
+        speaker=narrator
+        caption= _ "FORESTS"
+        image=terrain/forsest/deciduous-summer.png
+        message= _ "Forests can only be planted on grassland. The Loyalist and Outlaw factions can recruit Woodsmen, and the Woodsman and Poacher line will begin to automatically hunt for game in the forsest, producing gold dependent on thier level."
```

Comment:
typo with "their level"
I thought about hiding this text for non-human factions, but I think they can get Poachers too.

Reply:
Actually, that's a good point, I should add outlaws to the diplomacy options